skrf/util.py

```diff
@@ -46,6 +46,19 @@
 except NameError:
     basestring = (str, bytes)
 
+def network_array(v):
+    '''
+    Convenience function for creating properly formed network matrix vectors
+
+     Examples
+    -----------
+    >>> rseries = npy.ones(len(frequency))
+    >>> r_series_y = skrf.network_array([[1/rseries,  -1/rseries], [-1/rseries,  1/rseries]])
+    >>> n_rseries = skrf.Network.from_y(r_series_y)
+
+    '''
+    v = npy.array(v)
+    return v.swapaxes(0, 2).swapaxes(1, 2)
```

Comment:
I do agree that the name is bad, so that would need to change. 

My argument for keeping something like it is that it allows a user to input arbitrary networks. For example, suppose someone analytically determines  an admittance (Y) matrix for a network of resistors and capacitors, a utility like network_array is nice to have to generate the vector of matrices required by from_y. 

However, it's really not too hard to call val.swapaxes(0, 2).swapaxes(1, 2) on a numpy array so if you would prefer I can change my examples to do just that.  

My suggests would be to either:

1. rename network_array to something else (maybe _axes_swap_? )
2. use val.swapaxes(0, 2).swapaxes(1, 2) in the examples (probably introduces the least amount of clutter)